Library/Homebrew/utils/pypi.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module PyPI
+  module_function
+
+  def test_func
```

Comment:
Reminder to delete test function later :)

Reply:
😄